libgobuster/statehelpers.go

```diff
@@ -39,6 +40,36 @@ func InitState() State {
 	}
 }
 
+// ValidVerb ...
+func ValidVerb(verb string) bool {
```

Comment:
There is a potential we may want to be able to use non-valid verbs, or verbs that are only valid under certain protocols. Not a big issue for now though.

Reply:
Yes, just need to provide a clean way for the user to specify if there should be a content length/body for an invalid verb I guess